public/app/core/components/ForgottenPassword/ChangePassword.tsx

```diff
@@ -50,7 +55,12 @@ export const ChangePassword: FC<Props> = ({ onSubmit, onSkip }) => {
                 content="If you skip you will be prompted to change password next time you login."
                 placement="bottom"
               >
-                <LinkButton variant="link" onClick={onSkip} aria-label={selectors.pages.Login.skip}>
+                <LinkButton
+                  variant="link"
+                  onClick={onSkip}
+                  onKeyPress={onSkipKeypress}
+                  aria-label={selectors.pages.Login.skip}
+                >
```

Comment:
Sure, that's a good one.
Thanks @Clarity-89...would try it out!

Reply:
So, I tried out this approach and this was my findings:
- Link does not show any indication when it's been focused or hovered upon.
- Using the `enter` key to skip does some form validation before it performs the desired action which shouldn't be the case.

I understand that I can add some custom styling to solve the issue of the link not showing any effect when focused or hovered upon but considering the tradeoffs I mentioned above, do you think it's best for us to use `button` instead? @Clarity-89

Reply:
Hmm, that sounds like smth that needs to be addressed in the Button itself. Regarding using enter, does it help if you add `type="button"` to the button? Default button type is `submit` so that might be causing the issue.

Reply:
Yes! adding `type="button"` worked for me.
Thanks @Clarity-89 

I will just proceed to add a custom styling for hover and focus effects

Reply:
Actually I think instead of adding custom styles you could remove the rules overriding button's focus styles in grafana-ui: https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/Button/Button.tsx#L52, unless there's a reason they'd be kept @dprokop.

Reply:
Yeah...I agree.
@dprokop might have some thoughts on this.